### PR TITLE
Remove code for supporting older systems

### PR DESCRIPTION
**Pull request by uiryuu**
**Description:**
Remove code previously used for 10.11 - 10.14 systems. The major ones include a lot of colors, appearance, PIP and touchbar support.

### CODE REVIEW

File: iina/Base.lproj/PrefUIViewController.xib
Comment (low-batt) [inline iina/Base.lproj/PrefUIViewController.xib]: I guess Xcode makes odd changes like setting `y` to negative values due to constraints not being quite right? We have quite a few problems in the XIBs that should be fixed at some point.

File: iina/InitialWindowController.swift
Comment (low-batt) [inline iina/InitialWindowController.swift:174]: I recall something about needing to retain #available referencing 10.16 instead of 11 due to the need to support old SDKs. We might want to consider cutting off support for older Xcode versions, updating this, and allow use of the latest Swift features. But first we should merge a lot of the outstanding PRs before doing refactoring that can introduce merge conflicts.

**Review by low-batt — APPROVED** [review]
I reviewed all of the code changes. I pulled the PR, built IINA and tested under macOS Ventura and macOS Catalina.

Changes look good to me.

We should hold off doing more refactoring until we have merged more of the outstanding PRs to avoid merge conflicts.

**Comment by low-batt** [comment]
I may need to pull my approval. Hang on…

**Comment by low-batt** [comment]
I confirmed what I found is a regresison in `develop` and not due to this PR.

File: iina/Base.lproj/PrefUIViewController.xib
Comment (uiryuu) [reply iina/Base.lproj/PrefUIViewController.xib]: I didn't touch this file, should be an accident. Resetting this file...

**Comment by uiryuu** [comment]
Actually I will redo this PR in the future as the conflicts grows so quickly

**Pull request closed without merging**
